releases/mainnet/1.11.0/contracts.json

```diff
@@ -0,0 +1,306 @@
+{
+    "CHANGELOG": "0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F",
```

Comment:
extra spacing, line and missing flash killer ([diffcheck](https://www.diffchecker.com/rMHNsIH0))